<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemDescriptionDialog</class>
 <widget class="QDialog" name="ItemDescriptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>289</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Item/Description Editor</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="itemLabel">
     <property name="text">
      <string>Item:</string>
     </property>
     <property name="buddy">
      <cstring>itemEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="itemEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="locationLabel">
     <property name="text">
      <string>Location:</string>
     </property>
     <property name="buddy">
      <cstring>locationEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="locationEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="templateLabel">
     <property name="text">
      <string>Template:</string>
     </property>
     <property name="buddy">
      <cstring>templateCombo</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="templateCombo"/>
   </item>
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Description (optional):</string>
     </property>
     <property name="buddy">
      <cstring>descriptionTextEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="descriptionTextEdit"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>itemEdit</tabstop>
  <tabstop>locationEdit</tabstop>
  <tabstop>templateCombo</tabstop>
  <tabstop>descriptionTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemDescriptionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemDescriptionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
